Plugins/Player/Player.cpp

```diff
@@ -301,4 +302,14 @@ ArgumentStack Player::SetQuickBarSlot(ArgumentStack&& args)
     return stack;
 }
 
+ArgumentStack Player::GetBicFileName(ArgumentStack&& args)
+{
+    ArgumentStack stack;
+    if (auto *pPlayer = player(args))
+    {
+        Services::Events::InsertArgument(stack, std::string(pPlayer->m_resFileName.GetResRef()));
```

Comment:
This will fail in some circumstances- resrefs are 16 characters long and are not guaranteed to be null terminated. You could try this instead:

```c++
Services::Events::InsertArgument(stack, std::string(pPlayer->m_resFileName.GetResRef(), pPlayer->m_resFileName.GetLength()));
```